mozetl/shield/utils.py

```diff
@@ -17,7 +17,7 @@ def etl_job(sc, sqlContext, submission_date=None, save=True):
 
         transformed_pings = transform_func(sqlContext, pings)
 
-        if save:
+        if type(save) == bool and save is True:
```

Comment:
From what I understand, you don't want to have two separate datasets to process this dataset. If so, you should be taking the union of the two sets before writing them to disk.

On another note, the option you probably want is `mode="error"`. https://spark.apache.org/docs/latest/sql-programming-guide.html#save-modes